randovania/games/dread/json_data/Ghavoran.txt

```diff
@@ -179,7 +186,19 @@ Extra - asset_id: collision_camera_002
   * Extra - right_shield_entity: {EMPTY}
   * Extra - right_shield_def: None
   > Door to Left Entrance
-      Trivial
+      After Ghavoran - Gold Robot Fight
+  > Event - Gold Robot Fight
+      Any of the following:
+          After Elun - Release X Parasites
+          All of the following:
+              Before Elun - Release X Parasites
+              Any of the following:
+                  Fight Gold Robot
+                  All of the following:
+                      Speed Booster and Combat (Beginner)
+                      Any of the following:
+                          Missiles ≥ 29 and Shoot Missile
```

Comment:
I'm a little confused by the extra missile requirements when that should be covered by "Fight Gold Robot". I notice they're lower than the reqirements for "Fight Gold Robot", so is this a different robot then, like one of the silver ones? If so, we might want a different template or that

Reply:
This is coming in from the right entrance door, where you are able to get a shinespark onto the gold robot here, which knocks off almost half of its health. The shinespark can only be used once, and only on this robot

Reply:
This is the speed method i am referring to in the commit